src/components/menus/bus-menu.tsx

```diff
@@ -91,22 +127,45 @@ export const BusMenu: FunctionComponent<BusMenuProps> = ({
             }}
             onClose={onClose}
         >
+            {shortCircuitAvailability === OptionalServicesStatus.Up && (
+                <CustomMenuItem
+                    sx={styles.menuItem}
+                    onClick={handleClickRunShortcircuitAnalysis}
+                    selected={false}
+                    disabled={
+                        computationStarting ||
+                        oneBusShortcircuitAnalysisState === RunningStatus.RUNNING ||
+                        !isNodeEditable
+                    }
+                >
+                    <ListItemIcon>
+                        <BoltIcon />
+                    </ListItemIcon>
+
+                    <ListItemText
+                        primary={
+                            <Typography noWrap>
+                                <FormattedMessage id="ShortCircuitAnalysis" />
+                            </Typography>
+                        }
+                    />
+                </CustomMenuItem>
+            )}
+
             <CustomMenuItem
                 sx={styles.menuItem}
-                onClick={handleClickRunShortcircuitAnalysis}
+                onClick={() => handleTrip()}
                 selected={false}
-                disabled={
-                    computationStarting || oneBusShortcircuitAnalysisState === RunningStatus.RUNNING || !isNodeEditable
-                }
+                disabled={!isNodeEditable}
             >
                 <ListItemIcon>
-                    <BoltIcon />
+                    <OfflineBoltOutlinedIcon />
```

Comment:
Can we check with PO that we want the same icon for trip and dynamic SC?

Reply:
This is the same icon as other trip actions